src/components/elements/annotationSidePanel/components/AnnotationPolicyText.tsx

```diff
@@ -0,0 +1,135 @@
+                                <li>
+                                    To create annotations, users must have a valid ORCID identifier.
+                                </li>
+                                <li>
+                                    We recommend that users add their institutional email to their ORCID record.
+                                </li>
+                            </ul>
+                        </Col>
+                    </Row>
+                    {/* Visibility of annotations */}
+                    <Row className="mt-2">
+                        <Col>
+                            <p className="fs-3 fw-bold">
+                                Visibility of Annotations
+                            </p>
+
+                            <ul>
+                                <li>
+                                    All annotations are public and can be viewed by anyone.
+                                </li>
+                                <li>
+                                    No authentication is required to view annotation records.
+                                </li>
+                                <li>
+                                    Annotations are also accessible via our API without authentication.
+                                </li>
+                            </ul>
+                        </Col>
+                    </Row>
+                    {/* Machine annotations */}
+                    <Row className="mt-2">
+                        <Col>
+                            <p className="fs-3 fw-bold">
+                                Machine Annotations
+                            </p>
+
+                            <ul>
+                                <li>
+                                    Machine agents and machine annotation services must undergo review and verification before being added to the Machine Annotation Service.
+                                </li>
+                            </ul>
+                        </Col>
+                    </Row>
+                    {/* Archiving annotations */}
+                    <Row className="mt-2">
+                        <Col>
+                            <p className="fs-3 fw-bold">
+                                User Requirements
+                            </p>
+
+                            <ul>
+                                <li>
+                                    Users can archive their own individual annotations.
+                                </li>
+                                <li>
+                                    We do not delete annotations. Archived annotations are still accessible via the API;
+                                    they are just not visible with the latest digital specimen version.
+                                </li>
+                                <li>
+                                    [add examples of archived annotation when we have one; we decided to use the term archive
```

Comment:
probably shouldn't publish a todo on our site, even in test

Reply:
@sharifX

Reply:
yes, please remove this part.